synthtool/gcp/templates/node_library/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+Thanks for stopping by to let us know something could be better!
+
+Please run down the following list and make sure you've tried the usual "quick
+fixes":
+
+  - Search the issues already opened: https://github.com/googleapis/nodejs-storage/issues
```

Comment:
@crwilcox is there a good way to template out the url for the repo?

Reply:
`{{variable_name}}` as long as `variable_name` is passed in `synth.py`.

Reply:
So like you used `{{package_name}}` below, you can use `{{package_name}}` here if it can be used for the URL or you can use `{{owner}}/{{repo}}`.

Reply:
https://github.com/GoogleCloudPlatform/synthtool/blob/master/synthtool/gcp/common.py#L30 will need a new arg. Probably adding `repository_name` would be sufficient as package name will look like `@google-cloud/packagename` You may want to include owner as @theacodes suggested though. We have at least one repository not under googleapis.